lib/features/showcase/presentation/pages/scenario_page.dart

```diff
@@ -36,12 +38,11 @@ class _ScenarioPageState extends State<ScenarioPage> {
         actions: [
           TextButton.icon(
             onPressed: _toggleStyle,
-            icon: Icon(
-              _currentStyle == WidgetStyle.material
-                  ? Icons.apple
-                  : Icons.android,
-            ),
-            label: Text(_currentStyle.title),
+            icon: Icon(switch (nextStyle) {
+              WidgetStyle.material => Icons.android,
+              WidgetStyle.cupertino => Icons.apple,
+            }),
+            label: Text(nextStyle.title),
```

Comment:
Previously the icon would be the next style and the text was the current style. This makes both the text and icon the next style.